src/components/table/OcTableFiles.vue

```diff
@@ -65,9 +66,9 @@
       <oc-resource-size :size="item.size" />
     </template>
     <template #owner="{ item }">
-      <oc-avatars
```

Comment:
This looks like you haven't properly rebased your PR - the `oc-avatar-group` has been replaced by a (new) `oc-avatars` component which expects an `items` prop and not the (old) `users`